gateway/1.4/modules/ROOT/nav.adoc

```diff
@@ -1,7 +1,6 @@
 * xref:index.adoc[]
 * xref:flex-architecture-overview.adoc[]
 ** xref:flex-architecture-basic-deployments.adoc[]
```

Comment:
flex-gateway-k8-overview.adoc[] removed and content separated out to the reg/run sections for k8 and openshift